.github/workflows/docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Deploy API Documentation
+
+on:
+  push:
+    tags:
+      # Match 1.0.0 but not 1.0.0-rc1 or other suffixes
+      - '[0-9]+\.[0-9]+\.[0-9]'
+      - '![0-9]+\.[0-9]+\.[0-9]-[a-z]*'
+      - '![0-9]+\.[0-9]+\.[0-9]-[A-Z]*'
+
+jobs:
+  generate_docs:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Copy docs to docs/dokka directory
+        run: |
+          git fetch origin docs:docs
+          git checkout docs
+          mkdir -p docs/dokka || true
+          cp -R current/ docs/dokka/current
+          cp -R history/ docs/dokka/history
+          git checkout ${{ github.ref_name }}
+      - name: Generate Dokka Documentation
+        run: ./gradlew dokkaHtmlMultiModule
+      - name: Update Docs Branch
+        run: |
+          git checkout docs
+          cp -R docs/dokka/history .
+          cp -R docs/dokka/current .
+          git add history
+          git add current
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```

Comment:
Is that ok? Do we have some other values we could use here?

Reply:
🤷‍♂️ I don't have an answer. In fact, I have a question just out of curiosity. Can we use any mail there?